pkg/kruize/getPerformanceProfile.go

```diff
@@ -24,27 +24,65 @@ func NewProfileManager(profile_name string) *profileManager {
 	return pm
 }
 
-func (p *profileManager) GetPerformanceProfileInstance(clusterName string, namespace string,
-	workloadName string, containerName string) model.Perf_profile {
+//gets perfprof per container and returns query and name map
+func (p *profileManager) GetPerformanceProfileInstanceMetrics(clusterName string, namespace string,
+	workloadName string, containerName string, measurementDur string) []model.Metrics {
```

Comment:
Suggestion.  Consider creating an object for each workload instance.
```go
type WorkloadInstance struct {
  clusterName    string
  namepace       string
  name           string
  containerName  string
  measurementDur  string    // Why not time.Duration ?
}

Reply:
1. This same object can be enhanced to include the memory and cpu data.
2. The `replaceTemplate` function below becomes.
    ```go
    func (w *WorkloadInstance) replaceTemplate() {
    ...
    }
    ```